quickstarts/fivetran/config.yml

```
@@ -0,0 +1,31 @@
+slug: fivetran-integration
+title: Fivetran Destination Monitoring
+
+description: |
+  Monitor the status of your Fivetran Connectors to gain visibility on broken connectors.
+
+summary: |
+  Monitor Fivetran sync history and detect broken connectors.
+
+level: Community
+
+authors:
+  - Michael Young
+
+keywords:
+  - fivetran
+  - connector
+  - pipeline
+
```

```suggestion
icon: logo.svg
```
Update the logo file name here